source/_posts/2019-03-08-shipit-xiv.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "ShipIt XIV"
+author: Caitie Oder
+date: 2019-03-08
+tags: [culture, engineering, hackathon, shipit]
+---
+
+Readily awaiting the winter storm racing towards Kansas City, ShipIt Day XIV teams began preparing their ideas and tactics to compete in the 24-hour hackathon competition. Although the snowy weather may have kept participants between the warm walls of our Innovations Campus, the winter storm did not disturb the initiative and enthusiasm of the fourteen participating teams.
+It seems a bit daring to voluntarily bear the winter weather, stay up for 24-hours, complete a project start to finish, and place all other responsibilities on hold. Nevertheless, ShipIt Day always is bursting with Cerner associates prepared to take on the challenge.  *help this sentence please* Check out the video below to hear from ShipIt Day participants and supporters as to why ShipIt Day has become an engineering-wide program that supports Cerner’s development culture.
```

Comment:
> It seems a bit daring to voluntarily bear the winter weather, stay up for 24-hours, complete a project start to finish, and place all other responsibilities on hold.

I think it would be good to exclude this statement. While the time period is over 24 hours, we don't ask people to stay up for 24 hours, nor do we recommend it. It is just that we have the deadline set one day out, use the time you want in that window.

Reply:
Excluded introductory comment. Changed to suggestion below.